compose/src/main/java/com/bottlerocketstudios/compose/resources/Dimens.kt

```diff
@@ -73,6 +77,8 @@ val sw360Dimensions = Dimensions(
     grid_5 = 40.dp,
     grid_5_5 = 44.dp,
     grid_6 = 48.dp,
+    grid_12_5 = 96.dp,
```

Comment:
@VanyaHuamanBottleRocket Shouldn't this be `100` (12.5*8)?

Reply:
oops. lol changing